### DISCUSSION TITLE
Will it work with Laravel Nova?

### DISCUSSION THREAD

**Discussion by Tachii** [abr4xas/clarity-laravel#8] [Q&A] [answered]
I develop as B2B SaaS product and my target audience is internal team of admins, who operate in admin pane running on Laravel Nova.

It would be usefull for me to be able to see how they are using it, and what they use the most/the least etc.

So my question is - can I use it inside Laravel Nova?

https://nova.laravel.com

**Comment by abr4xas** [comment, marked as answer]
Hey @Tachii, tbh, I’m not sure. Perhaps you'll need to publish the stubs and use the components in your views.

https://nova.laravel.com/docs/customization/stubs.html

**Reply by Tachii** [reply]
Thanks, I'll give it a try and will let you know
